render_pdf/src/make_latex.py

```diff
@@ -0,0 +1,92 @@
+from typing import Union
+import parser
+import os
+import jinja2
+
+# TODO probably best to use pydantic for a config template for the dict below. 
+data = {
+        'language' : "sanskrit",
+        'title' : "",
+        'author' : "",
+        'body' : [],
+        'analysis': []
+    }
+
+os.chdir("../tex_files")
+latex_jinja_env = jinja2.Environment(
+        block_start_string = '((*',
```

Comment:
I'm curious how you chose this syntax? `(((` in particular is quite hard for me to read ...